.github/images/run_tests.sh

```diff
@@ -1,7 +1,5 @@
 #!/bin/bash
 
-bundle install
```

Comment:
Are we removing this, because we now have the cache in the container and are doing this in the `build_container` workflow?

Reply:
It is done in [Dockerfile](https://github.com/theforeman/foreman_virt_who_configure/blob/eda1496e3f28613e5b1e681b76b88b01da1c7a5e/.github/images/Dockerfile#L32).